lib/Common/Core/FaultInjection.cpp

```diff
@@ -1228,16 +1228,20 @@ namespace Js
         GetModuleHandleEx(GET_MODULE_HANDLE_EX_FLAG_FROM_ADDRESS, reinterpret_cast<LPCTSTR>(ip), &mod);
         offset = ip - (uintptr_t)mod;
         auto& faultModule = modulePath;
-        GetModuleFileName(mod, faultModule, MAX_PATH);
-        fwprintf(stderr, _u("***FI: Exception: %08x, module: %s, offset: 0x%p\n"),
-            ep->ExceptionRecord->ExceptionCode, faultModule, (void*)offset);
+        if (GetModuleFileName(mod, faultModule, MAX_PATH))
+        {
+            fwprintf(stderr, _u("***FI: Exception: %08x, module: %s, offset: 0x%p\n"),
+                ep->ExceptionRecord->ExceptionCode, faultModule, (void*)offset);
```

Comment:
The buffer faultModule that GetModuleFileName fills is not used after the fwprintf, so I guess it's fine to only skip fwprintf and continue, when it fails.